lib/Model/NLUAction.dart

```diff
@@ -2,8 +2,9 @@ import 'package:json_annotation/json_annotation.dart';
 
 @JsonSerializable(nullable: true)
```

Comment:
Got the following error while trying to build this is the following command `flutter pub run build_runner build --delete-conflicting-outputs`

`The class `ActionType` has no default constructor.
package:untitled3/Model/NLUAction.dart:4:6`


The error disappeared when I commented out the @JsonSerializable(nullable: true)

Reply:
@JsonSerializable(nullable: true) expects every Class to define a default constructor (but this ENUM does not have one).

Reply:
Fixed the issue The class ActionType has no default constructor